selfservice/strategy/oidc/provider_apple.go

```diff
@@ -146,3 +147,21 @@ func decodeQuery(query url.Values, claims *Claims) {
 		}
 	}
 }
+
+var _ IDTokenVerifier = new(ProviderApple)
+
+func (a *ProviderApple) Verify(ctx context.Context, rawIDToken string) (*Claims, error) {
```

Comment:
One limitation of this implementation is that we don't request ATs & RTs during this flow, so if users want to use them, on behalf of their users, this wouldn't work. 

I am not sure what use cases there are for using an AT against Apple's API, but I'd suggest we can implement the `authCode` flow at a later time. It should be trivial, but [doesn't work with the way Expo has implemented Apple authentication](https://github.com/expo/expo/issues/16162), as it requests the tokens using the client ID `host.exp.Exponent`.

Reply:
Please document this as a caveat.

Reply:
Done in the Docs PR.